static/app/views/organizationGroupDetails/groupReplays/groupReplays.spec.tsx

```diff
@@ -0,0 +1,460 @@
+            projectId: '2',
+            startedAt: '2022-09-15T06:50:03+00:00',
+            urls: [
+              'https://dev.getsentry.net:7999/organizations/sentry-emerging-tech/replays/',
+              '/organizations/sentry-emerging-tech/replays/?project=2',
+            ],
+            user: {
+              id: '147086',
+              name: '',
+              email: '',
+              ip_address: '127.0.0.1',
+              displayName: 'testDisplayName',
+            },
+          },
+          {
+            countErrors: 4,
+            duration: 400,
+            finishedAt: '2022-09-21T21:40:38+00:00',
+            id: 'b05dae9b6be54d21a4d5ad9f8f02b780',
+            projectId: '2',
+            startedAt: '2022-09-21T21:30:44+00:00',
+            urls: [
+              'https://dev.getsentry.net:7999/organizations/sentry-emerging-tech/replays/?project=2&statsPeriod=24h',
+              '/organizations/sentry-emerging-tech/issues/',
+              '/organizations/sentry-emerging-tech/issues/?project=2',
+            ],
+            user: {
+              id: '147086',
+              name: '',
+              email: '',
+              ip_address: '127.0.0.1',
+              displayName: 'testDisplayName',
+            },
+          },
+        ],
+      },
+    });
+
+    // Mock the system date to be 2022-09-28
+    jest.useFakeTimers().setSystemTime(new Date('Sep 28, 2022 11:29:13 PM UTC'));
+
+    renderComponent();
+
+    await waitFor(() => {
+      expect(mockApi).toHaveBeenCalledTimes(1);
+    });
+
+    // Expect the table to have 2 rows
+    expect(screen.getAllByText('testDisplayName')).toHaveLength(2);
+
+    // Expect the first row to have the correct href
+    expect(screen.getAllByRole('link', {name: 'testDisplayName'})[0]).toHaveAttribute(
+      'href',
+      '/organizations/org-slug/replays/project-slug:346789a703f6454384f1de473b8b9fcc/?referrer='
+    );
+
+    // Expect the second row to have the correct href
+    expect(screen.getAllByRole('link', {name: 'testDisplayName'})[1]).toHaveAttribute(
+      'href',
+      '/organizations/org-slug/replays/project-slug:b05dae9b6be54d21a4d5ad9f8f02b780/?referrer='
+    );
+
+    // Expect the first row to have the correct duration
+    expect(screen.getByText('14hr 32min 26s')).toBeInTheDocument();
+
+    // Expect the second row to have the correct duration
+    expect(screen.getByText('6min 40s')).toBeInTheDocument();
+
+    // Expect the first row to have the correct errors
+    expect(screen.getByText('1')).toBeInTheDocument();
+
+    // Expect the second row to have the correct errors
+    expect(screen.getByText('4')).toBeInTheDocument();
+
+    // Expect the first row to have the correct date
+    expect(screen.getByText('14 days ago')).toBeInTheDocument();
+
+    // Expect the second row to have the correct date
+    expect(screen.getByText('7 days ago')).toBeInTheDocument();
+  });
+
+  it('Should sort by start time correctly', async () => {
```

Comment:
Kind of a nitpick, but I would rename this test, and the related ones below to be more like (but doesn't have to be exactly like):

```suggestion
  it('should request data sorted by startedAt when the Start Time column is clicked', async () => {
```